### ISSUE TITLE
Fix links to repos moved from edx org

### ISSUE THREAD

**Issue by timmc-edx** [edx/edx-arch-experiments#558]
A/C:

- [x] High-urgency references updated in edx and openedx repos, and 2U deployments unblocked
- [x] Remaining references updated or maintainers notified
- [x] Communication about possible effects on named releases

Context: https://discuss.openedx.org/t/please-update-your-git-urls-for-edx-platform-and-several-other-repos/12387

GitHub search for all URLs that should be updated: `(org:openedx OR org:edx) NOT is:archived /(?i)github\.com[^A-Za-z]+edx\/(edx-platform|cs_comments_service|XBlock|xblock-sdk|devstack|django-require)\b/` (based on repos of same name in both orgs)

Snippet for fixing these URLs in a repo:

`git ls-tree HEAD -r --full-tree | grep '^10' | cut -d$'\t' -f2 | while IFS= read -r 'path'; do sed -i 's%\(github\.com[^A-Za-z]\+\)ed[xX]/\(ed[xX]-platform\|cs_comments_service\|XBlock\|xblock-sdk\|devstack\|django-require\)%\1openedx/\2%g' -- "$path"; done`

**Comment by timmc-edx** [comment]
There are still other references, but they're lower priority and we can find and fix them as needed.
